lib/problems.js

```diff
@@ -0,0 +1,55 @@
+function balanced(string) {
```

Comment:
💫🌸 Impressive! You didn't need to implement this, but good work. I apologize for the confusion!